packages/tabs/src/TabsContainer.tsx

```diff
@@ -10,8 +10,12 @@ import * as React from 'react'
 import { useEffect, useState } from 'react'
 import styled from 'styled-components'
 import { Icon, Tooltip } from '@equinor/eds-core-react'
-import { home } from '@equinor/eds-icons'
 import { TabsProvider } from './TabsContext'
+import { home } from '@equinor/eds-icons'
+
+Icon.add({
+  home,
+})
```

Comment:
This should not be needed when you use the "data" prop with the icon on line 124

Reply:
removed